### ISSUE TITLE
url() doesn't work for DRF and Django >=1.10

### ISSUE THREAD

**Issue by yuroapps** [yourlabs/django-cities-light#156]
To implement this on DRF 3, the documentation (http://django-cities-light.readthedocs.io/en/stable-3.x.x/contrib.html#for-djangorestframework) says that the following url should be included:

`url(r'^cities_light/api/', include('cities_light.contrib.restframework3')),`

However, the include tries to import patterns, which apparently since Django 1.10, the patterns module has been removed (https://stackoverflow.com/a/38799716)

**Comment by max-arnold** [comment]
Could you please try the latest revision from master branch?

**Comment by mikanchu** [comment]
It would be nice if you can release a minor with the state of master. Thanks!

**Comment by jpic** [comment]
@mikanchu does that means the current master head works for you ?
@max-arnold any objection into making a minor release ?

**Comment by max-arnold** [comment]
@jpic No objections (if you plan to remove obsolete autocomplete-light contrib).

**Comment by mikanchu** [comment]
@jpic yes, it does. The use I am doing is not extensive but at least it builds which the pip version doesn't

**Comment by max-arnold** [comment]
@jpic Any progress with the release? Do you need any help?

**Comment by max-arnold** [comment]
@jpic ?
